native/packages/react-native-bpk-component-icon/readme.md

```diff
@@ -8,9 +8,20 @@
 npm install react-native-bpk-component-icon --save-dev
 ```
 
-<!-- // TODO ADD INSTRUCTIONS FOR ADDING THE PODSPEC / GRADLE SCRIPT TO THE iOS / Android PROJECT IN WHICH THIS IS NOW BEING USED! -->
+## Native usage
 
-## Usage
+### IOS
+
+Add the following to your `Podfile` and run `pod update`:
+
+`pod 'BpkIcons', :path => '../react-native/node_modules/react-native-bpk-component-icon'`
```

Comment:
@k0nserv could you please double check this?

Reply:
I think the pod name should match what you call the pod here. i.e if it's `BpkIcons` here then the podspec should be `BpkIcons.podspec` and the name in the podspec should be `BpkIcons`

Reply:
nice one, I thought I renamed it, and I obviously didn't!